# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: third_party/rust/wgpu-core-deps-windows-linux-android/.cargo-checksum.json
# Commit: b3109ad14685
# Full Hash: b3109ad14685624ada7861083ff5332aec9a7818
# Author: Andy Leiserson <aleiserson@mozilla.com>
# Date: 2025-10-07 17:00 +0000
# Description:
#   Bug 1992842 - Update wgpu to upstream 1072b878 (2025-10-06) r=webgpu-reviewers,supply-chain-reviewers,ErichDonGubler
# ==============================================================================

diff -r 1d3f000cf1f9 -r b3109ad14685 third_party/rust/wgpu-core-deps-windows-linux-android/.cargo-checksum.json
--- a/third_party/rust/wgpu-core-deps-windows-linux-android/.cargo-checksum.json	Tue Oct 07 16:51:44 2025 +0000
+++ b/third_party/rust/wgpu-core-deps-windows-linux-android/.cargo-checksum.json	Tue Oct 07 17:00:40 2025 +0000
@@ -1,1 +1,1 @@
-{"files":{"Cargo.toml":"0830990748013fbda96fe8a5b1c58be9fafd2c6680f025f50082be3817c10f90","README.md":"38086a02e134ac959bd061f1161c141a848a1b05a6bf31874035908217ad3eed","src/lib.rs":"a99034037e9c9ddf912f44c05b98af2c11e0ed0d09bb7cb69577826f46062ab9"},"package":null}
\ No newline at end of file
+{"files":{"Cargo.toml":"28ba3f78bab861041fa7ba583eff4967fce611252ca87d9eb364df7822f9cdae","LICENSE.APACHE":"a6cba85bc92e0cff7a450b1d873c0eaa2e9fc96bf472df0247a26bec77bf3ff9","LICENSE.MIT":"dc0d97139e8205818c703741c7be7cb3b96888bd5917b8d6fc6133731e403c21","README.md":"38086a02e134ac959bd061f1161c141a848a1b05a6bf31874035908217ad3eed","src/lib.rs":"a99034037e9c9ddf912f44c05b98af2c11e0ed0d09bb7cb69577826f46062ab9"},"package":null}
\ No newline at end of file